### PR TITLE
Try not to cull lines

### PR DESCRIPTION
**Pull request by phocks**
This should mean that lines aren't culled (in most cases). I noticed a line was still disappearing right now the bottom on that major zoom, so I slightly tweaked the FOV to hide it....... hopefully all good.

### CODE REVIEW

File: src/components/Graph/canvas.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ export default class Canvas {
     this.scene = new Scene();
     this.scene.background = new Color(0x5f6b7a);
     this.camera = new PerspectiveCamera(75, width / height, 0.1, 1000);
+    this.camera.fov = 60;
```
Comment (drzax) [inline src/components/Graph/canvas.js]: Why this change? (And if we're changing the field of view here , we might as well just do it in the constructor above.)

Comment (phocks) [reply src/components/Graph/canvas.js]: "I noticed a line was still disappearing right now the bottom on that major zoom, so I slightly tweaked the FOV to hide it" :)

And yep will do it in the constructor

Comment (phocks) [reply src/components/Graph/canvas.js]: OK done!
